fbrp/examples/02_multiproc/alice.py

```diff
@@ -2,7 +2,7 @@
 import time
 
 i = 0
-p = a0.Publisher("imu/image")
+p = a0.Publisher("some/topic")
```

Comment:
I forgot to ask, are topics hierarchical ? Say `a/b/c` means `c` under `b` which is under `a` or some such ? Not a feature request if it is not, just asking for my understanding